test/topology/conftest.py

```diff
@@ -190,3 +192,7 @@ async def random_tables(request, manager):
     failed = request.node.stash[FAILED_KEY]
     if not failed and not await manager.is_dirty():
         tables.drop_all()
+
+@pytest.fixture(scope="function")
+def mode(request):
+    return request.config.getoption('mode', 'release')
```

Comment:
So this will by default return `release` if no mode was specified? I wonder if it's correct to do so. Maybe we should return optional (or empty string if mode was specified). Or require that `mode` is always present (isn't it?)